src/query/storages/common/cache-manager/src/cache_manager.rs

```diff
@@ -104,8 +106,11 @@ impl CacheManager {
                 CompactSegmentInfoMeter {},
                 "segment_info",
             );
-            let bloom_index_filter_cache =
-                Self::new_item_cache(config.table_bloom_index_filter_count, "bloom_index_filter");
+            let bloom_index_filter_cache = Self::new_in_memory_cache(
+                config.table_bloom_index_filter_count * std::mem::size_of::<Xor8Filter>() as u64,
```

Comment:
afraid that we can not simply multiply by the size of type `Xor8Filter`, please see the comment of 
 
 `impl Meter<String, Arc<Xor8Filter>> for BloomIndexFilterMeter`.
 
 
 **maybe** we need to introduce new config item `table_bloom_index_filter_size`, which user can specify the max size (in bytes) of cached bloom index filters? and deprecate this config option?.
 
 but that may introduce incompatible changes, if user has non-default  (explicit) config item of `table_bloom_index_filter_count`.

Reply:
introduce new_item_cache_with_meter, it use meter instead of count to meter cache size